source/information-security/certificate-automation.html.md.erb

```diff
@@ -23,6 +23,15 @@ unexpected certificate expiry.
 
 ## Manual certificate requests
 
+Where the use of LetsEncrypt is suitable, New requests for certificates should be submitted to the Platform Operations team by raising a DTSPO Jira ticket containing the following information:
```

Comment:
I think we should re-word this a bit.

If your site is being hosted on:
(Link the below services to their product docs)
 * Azure FrontDoor
 * Azure App Service
 * Azure CDN
Then you must use the 'Microsoft managed certificates'

Otherwise please raise a request in #platops-help (there should be a redirect link used for this)

Reply:
![Screenshot 2021-06-29 at 13 52 51](https://user-images.githubusercontent.com/54357991/123800692-765d4f80-d8e1-11eb-9768-98a2b5d80f34.png)

Reply:
better can you push it though. remove the Once complete raise a request in platopshelp bit.

remove the CSR form bit as that will move to ops-runbooks

Reply:
![Screenshot 2021-06-30 at 10 51 37](https://user-images.githubusercontent.com/54357991/123940702-56856480-d991-11eb-9177-e8bc1267f321.png)
pushed + screenshot